packages/side-nav/src/SideNavLink.tsx

```diff
@@ -4,23 +4,26 @@ import { useLinkComponent, mapSpacing } from '@ag.ds-next/core';
 import { boxPalette, fontGrid, outline } from '@ag.ds-next/core';
 
 import { useLinkListDepth } from './context';
-import { localPalette } from './utils';
+import { localPalette, visuallyHiddenStyles } from './utils';
 
 export type SideNavLinkType = {
 	active?: boolean;
+	activePath?: string;
 	children?: ReactNode;
 	href: string;
 	label: string | JSX.Element;
 };
 
 export const SideNavLink = ({
 	active,
+	activePath,
 	children,
 	href,
 	label,
 }: SideNavLinkType) => {
 	const depth = useLinkListDepth();
 	const Link = useLinkComponent();
+	const isGroupActive = activePath?.includes(href);
```

Comment:
This can cause more than one item in a set to appear active.
Eg. `/packages/text` and `/packages/text-link` will both match here.